activation/2.0/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Jakarta Activation 2.0 (milestone)"
+title: "Jakarta Activation 2.0"
 date: 2020-06-15
```

Comment:
I would adjust this date to be closer to the expected Ballot success date, which I would say is August 10 at the earliest at this point.